### PR TITLE
[perf] Add more perf sqls

### LINKED ISSUE #1347: [Performance Test] Improve charset display in performance.html
**Summary**
seems like report html has some charset display issues, and could be prettified
https://test-1306375239.cos.accelerate.myqcloud.com/report/1334/b7b370b3e8336ae007a1fa143939675443c9b2f0/36b91cc0-5a19-4256-9644-6c71011831a4/performance.html

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

- Add more perf sqls
- Fix perf htmls

## Changelog

- Improvement

## Related Issues

Fixes #1347

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: tests/perfs/compare.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def load_config():
             <th>{},&nbsp;s</th>
             <th>{},&nbsp;s</th>
             <th>Ratio of speedup&nbsp;(-) or slowdown&nbsp;(+)</th>
-            <th>Relative difference (after_score&nbsp;−&nbsp;before_score) / before_score</th>
+            <th>Relative difference (after_score&nbsp;&minus&nbsp;before_score) / before_score</th>
```
Comment (PsiACE) [inline tests/perfs/compare.py:106]: `&minus;`

Comment (BohuTANG) [reply tests/perfs/compare.py:106]: Another confused:
```
Ratio of speedup (-) or slowdown (+)
```

is it
```
Ratio of speedup (+) or slowdown (-)?
```
cc @ZhiHanZ

Comment (sundy-li) [reply tests/perfs/compare.py]: > `&minus;`

Good catch.

**Review by ZhiHanZ — APPROVED** [review]
data volume seems to be fine, and LGTM

**Pull request merged**